src/Avalonia.Controls/Primitives/SelectingItemsControl.cs

```diff
@@ -639,7 +639,7 @@ private static IEnumerable<object> GetRange(IEnumerable items, int first, int la
         /// </summary>
         /// <param name="items">The items collection.</param>
         /// <param name="desired">The desired items.</param>
-        private static void SynchronizeItems(IList items, IEnumerable<object> desired)
+        internal static void SynchronizeItems(IList items, IEnumerable<object> desired)
```

Comment:
I was not sure where to put it, do you follow any file layout guidelines in Avalonia?

Reply:
It's not a strict ordering, but I usually use the order `'public`, `protected`, `internal`, `private`, i.e. from most public to least public.

I know there are many places where we don't follow that, mostly because of edits. However, if we can keep it tidy i'd prefer it and an `internal` modifier between `private` methods just sets my senses tingling ;)

Reply:
I see, at work I use the R# file layout that is automatically applied on their cleanup reformat. Makes life a bit easier when you don't have to think about it :)

Reply:
Yeah, I really wish there was a `gofmt` for csharp :/